uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/JobRequestScheduler.java

```diff
@@ -174,7 +171,7 @@ public void deployProjectRequest(final DeployProjectRequest jobRequest) {
                                          params));
     }
 
-    public void createOrganizationalUnitRequest(final SpaceRequest jobRequest) {
+    public void createSpaceRequest(final SpaceRequest jobRequest) {
```

Comment:
Is in this method used right operation? In this method is call command to create Org unit.

Reply:
The OrganizationalUnit is getting deleted and at the moment it is basically a Space.